rllib/policy/eager_tf_policy.py

```diff
@@ -332,8 +332,15 @@ def compute_actions(self,
             if action_sampler_fn is not None:
                 state_out = []
                 action, logp = action_sampler_fn(
-                    self, self.model, input_dict, self.observation_space,
-                    self.action_space, explore, self.config, timestep)
+                    self,
+                    self.model,
+                    input_dict,
+                    self.observation_space,
+                    self.action_space,
+                    explore,
+                    self.config,
+                    timestep=timestep
+                    if timestep is not None else self.global_timestep)
```

Comment:
I think this was a bug where we never used the global timestep when eager was on.